app/api/v2/models.py

```diff
@@ -0,0 +1,298 @@
+import psycopg2
+import os
+
+
+class Dbase():
+    def __init__(self):
+        self.db_name = 'politico'
+        self.db_host = 'localhost'
+        self.db_user ='postgres'
+        self.db_password ='Yunis1500'
+        self.conn = None
+
+    def connection_to_Dbase(self):
+        if os.getenv("APP_SETTINGS") == "development":# get from app settings:development
+            try:
+                self.conn= psycopg2.connect(
+                    database= self.db_name,
+                    host=self.db_host,
+                    user=self.db_user,
+                    password=self.db_password
+                )
+                print("connection successful")
+            except Exception as e:
+                print( e, "can't connect to database")
+
+            return self.conn
+    
+    def create_tables(self):
+        tables= [
+            """
+            CREATE TABLE IF NOT EXISTS users(
```

Comment:
This can be improved. You are CREATE TABLE IF NOT EXISTS. You can improve this further